### PR TITLE
KO: Update segmentation.ipynb from docs repo.

### PR DESCRIPTION
**Pull request by rickiepark**
Update from recent changes in segmentation.ipynb 
Colab checked.
Thanks. :)

### CODE REVIEW

**Comment by lamberta** [comment]
Thanks, Haesun

To make the diff easier to read, can you run `segmentation.ipynb` through [nbfmt.py](https://github.com/tensorflow/docs/blob/master/tools/nbfmt.py) ?

```
./tools/nbfmt.py --ignore_warn path/to/notebook.ipynb
```

**Comment by rickiepark** [comment]
Sure, Thanks. :)

**Comment by rickiepark** [comment]
Hi, @jaketae 
Could you please review this PR? 😃

**Comment by jaketae** [comment]
Hi @rickiepark,

Thank you very much for the PR. I have very little to suggest, but here are my comments:

* Remove `%tensorflow_version 2.x` and other outdated import commands
* Hide raw links in lines 369 and 394 with `[text_displayed](link)` markdown formatting
* In the function `unet_model(output_channels)`, `last = tf.keras.layers.Conv2DTranspose` declaration should be at the end, not the front (refer to the [original notebook](https://github.com/tensorflow/docs-l10n/blob/3673f411b48e61adcbb63f81097bfdccf493dca0/site/ko/tutorials/images/segmentation.ipynb))
*  For the loss function, use `loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True)` instead of `loss='sparse_categorical_crossentropy'`

Normally, I would try to comment on the PR, but I wasn't able to (I think this is because you're updating a notebook, not submitting a new one). Anyhow, I hope you find my comments helpful.

Thank you again for the PR. Have a great weekend!

**Comment by jaketae** [comment]
LGTM, thank you very much as always @rickiepark!

@lamberta Is running nbmft necessary for easier diff-reading? If not, I think we're good to merge.

**Comment by rickiepark** [comment]
Thank you sooooo much @jaketae 
I update the notebook by nbfmt

I'll go for next notebook(save_and_load.ipynb). 😄

**Comment by jaketae** [comment]
@rickiepark Not at all, you're doing all the work! Thank YOU for translating and reviewing other notebooks. I'll try to work on some PRs this week as well to share the burden. Thanks again 🙏

**Comment by rickiepark** [comment]
Thank you @jaketae , I'll review the PRs you sumitted as well. :)
